extra/onnx_ops.py

```diff
@@ -37,6 +37,12 @@ def BatchNormalization(X, scale, B, input_mean, input_var, epsilon=1e-05, moment
   else:
     invstd = (input_var + epsilon)**-0.5
     return X.batchnorm(scale, B, input_mean, invstd)
+  
+def LayerNormalization(x: Tensor, scale, bias, axis=-1, epsilon=9.999999747378752e-06, stash_type=1):
```

Comment:
Why the weird value for epsilon? 1e-05 instead?

Reply:
the docs I was referencing used that as the default, will switch to 1e-05